docs/stories/components/Checkbox.stories.js

```diff
@@ -62,10 +62,7 @@ function CheckboxStory() {
         <ul>
           {Object.keys(checkboxes).map(current => {
             return (
-              <li
-                key={current}
-                style={{ marginBottom: 1, paddingTop: 18, paddingLeft: 15 }}
-              >
+              <li key={current} style={{ paddingTop: 18, paddingLeft: 15 }}>
```

Comment:
We should avoid these kinds of object literal to improve the performance of the application. This is similar to the anonymous functions. Object literals don't have a persistent memory space, so the component will need to allocate a new location in memory for this object whenever the component re-renders.

Reply:
You don't need to fix them for the moment but we need to keep in mind those best practices.

Reply:
I agree plus it seems that the baseline is not kept here